src/types/OpsLevelData.ts

```diff
@@ -28,7 +53,10 @@ export interface OpsLevelServiceData {
         overallLevel: OverallLevel,
         categoryBreakdown: Array<LevelCategory>,
       },
-      serviceStats: {
+      serviceStats?: {
+        scorecards?: {
+          nodes?: Array<ScorecardStats>
+        },
```

Comment:
We should really go through these interfaces at some point since there are a lot of things that are listed in the API as optional but are listed here as non-optional.

I corrected some of them related to scorecards but it's prone to risk.